lua/gluatest/expectations/negative.lua

```diff
@@ -24,6 +24,15 @@ return function( subject, ... )
     end
     expectations.equal = expectations.eq
 
+    function expectations.aboutEqual( comparison )
+        local tolerance = args[1] or 0.00001
+        local difference = math.abs( subject - comparison )
+
+        if difference <= tolerance then
+            i.expected( "to not be within '%s' of '%s' - found a difference of '%s'", tolerance, comparison, difference )
+        end
+    end
+
```

Comment:
If you're going to have a `.eq` alias for `.equal`, you should have a `.aboutEq` alias.

Or get rid of `.eq` because it's icky. (More like `.ew`)

Reply:
I see `eq` as deprecated. I'll mark it as such actually

Reply:
If it's depreciated, the function should be `.equal` and `.eq` should be the alias (Or, again, remove it)